protocols.csv

```diff
@@ -10,4 +10,5 @@ code	size	name
 421	V	ipfs
 480	0	http
 443	0	https
-444	10	onion
\ No newline at end of file
+477	0	websockets
```

Comment:
Is this written down somewhere? Cause we should add these things to the multiaddr spec first before changing implementations all over the place